root/admin/LockedUsernameUnlock.js

```diff
@@ -35,7 +35,7 @@ const LockedUsernameUnlock = ({
       <form method="post" name="confirm">
         <FormCsrfToken form={form} />
         <FormSubmit
-          label="Yes, I’m sure"
+          label="Yes, I am sure"
```

Comment:
We might as well put this into l() then since it's an existing translation?

Reply:
It’s an admin form though, we shouldn’t really translate it.

Reply:
As you prefer :) I think we're very inconsistent right now with translating some and not others, so it'd be good to eventually drop those if we decide not to translate admin forms as a rule.

Reply:
Maybe we should have a dummy `l_admin` that does nothing but prevents messages from being added to POT files.